src/java/org/apache/cassandra/index/sai/disk/v2/sortedterms/SortedTermsReader.java

```diff
@@ -105,19 +105,34 @@ public SortedTermsReader(@Nonnull FileHandle termsData,
 
     /**
      * Returns the point id (ordinal) of the target term or the next greater if no exact match found.
+     * @param term
+     * @return
+     */
+    public long getNextPointId(@Nonnull ByteComparable term)
+    {
+        return getPointId(term, false);
+    }
+
+    /**
+     * Returns the point id (ordinal) of the target term.
      * If reached the end of the terms file, returns <code>Long.MAX_VALUE</code>.
      * Complexity of this operation is O(log n).
      *
      * @param term target term to lookup
      */
-    public long getPointId(@Nonnull ByteComparable term)
+    public long getExactPointId(@Nonnull ByteComparable term)
+    {
+        return getPointId(term, true);
+    }
+
+    private long getPointId(@Nonnull ByteComparable term, boolean exactMatch)
```

Comment:
not intuitive to me what to expect from non-exact-match mode, could we flip the meaning and call it prefixMatch perhaps?  (assuming that's what it does)

Reply:
ah, that's actually made clear by the calling methods.  feel free to ignore this

Reply:
I'll address that comment in https://github.com/datastax/cassandra/pull/787 (I merged that branch into this one)